README.md

```diff
@@ -130,3 +130,7 @@ Then restart logic. If everything works and starts up again you can delete the c
 ## Building - VST2
 
 If you want to build VST2 versions of the plug-in, set the environment variable VST2SDK_DIR to the location of the SDK prior to building.
+
+An example of setting the environment variable `VST2SDK_DIR` would be:
+
+```export VST2SDK_DIR=~/programming/VST_SDK_2.4```
```

Comment:
Might add a note that this needs to be set prior to running `premake5 xcode4`, which generates projects (and is part of `build-osx.sh`).